salesagility/SuiteCRM
# WIP Fix/metting and call validation through calendar

## Description



This fix adds in validation to the meetings/call creation through the calendar form popup.

## Motivation and Context

When creating a meeting/call within the calendar popup the validation is not in place, therefor these can be created without titles, without dates and without duration's.

## How To Test This

1) Have the calendar dashlet available
2) Select one or more time slots to allow the meetings/calls creation popup to appear
3) Fill out either or both of these forms
4) Try missing out required fields, e.g. Title, Start Date, End Date etc
5) Also try adding in a negative duration, e.g. End date before the Start date

## Types of changes


### Final checklist
